package.json

```diff
@@ -47,6 +58,8 @@
     "@rollup/plugin-commonjs": "^11.0.2",
     "@rollup/plugin-node-resolve": "^7.1.1",
     "@types/jest": "^25.1.2",
+    "@vue/test-utils": "^1.0.4",
+    "babel-core": "^7.0.0-bridge.0",
```

Comment:
This should not be needed - the 7.0.0-bridge.0 release is for projects that use babel 6

Reply:
Hey @joeldenning, it stills lacks the update and unmount lifecycles which I'm working on right now. I'll also take a look onto this package.json file, and squashing some commits to avoid that huge historic.

Reply:
This repository uses squash and merge, which means that it doesn't matter how many commits your branch has. So you don't have to worry about that.